dali/test/python/unittest_failure.cfg

```diff
@@ -0,0 +1,15 @@
+[unittest]
```

Comment:
Do we need this config?

Reply:
So I can have a test case starting with `failure_` that won't be picked up by anyone that wants to run all the tests.
I also want to test the typical unit-testing scenario, that's why I want a test file and nose invocation.